GameDataParser/Parsers/MapEntityParser.cs

```diff
@@ -159,6 +218,16 @@ private static CoordS ParseCoord(string value)
             );
         }
 
+        private static CoordB ParseCoordB(string value)
```

Comment:
Can this method live inside `CoordB`? Seems like there's already methods there for creating new Coords from existing data